bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/handler/DoorbellHandler.java

```diff
@@ -145,23 +133,23 @@ public void dispose() {
 
     // Callback used by listener to get Doorbird host name
     public @Nullable String getDoorbirdHost() {
-        return config.doorbirdHost;
+        return getConfigAs(DoorbellConfiguration.class).doorbirdHost;
```

Comment:
What is the benefit over caching the config (like it was before)?

Reply:
None whatsoever. 🙄 I don't know what I was thinking. It's back to what it was before.